engine/table/src/main/java/io/deephaven/engine/table/impl/QueryTable.java

```diff
@@ -636,6 +636,10 @@ public Table tail(final long size) {
 
     @Override
     public Table headPct(final double percent) {
+        if (percent < 0 || percent > 1) {
+            throw new IllegalArgumentException(
+                    "headPct percentage provided should be between [0,1]: percent=" + percent);
```

Comment:
This function can be called from python and will have a different parameter name after mapping.  Use the core concept rather than the parameter name in the error.  I would also remove the word `provided` and go with something like this:

`percentage of rows must be between [0,1]: percent=`

Reply:
When we achieve clarity on the interval, make sure the interval in the exception matches (i.e. this might be `(0,1)`)